Rock.Encryption/Crypto.cs

```diff
@@ -197,47 +197,28 @@ public static Task<byte[]> DecryptAsync(byte[] cipherText, object keyIdentifier
         }
 
         /// <summary>
-        /// Gets an instance of <see cref="IAsyncEncryptor"/> for the provided encrypt key.
-        /// </summary>
-        /// <returns>An object that can be used for encryption operations.</returns>
-        public static IAsyncEncryptor GetAsyncEncryptor()
-        {
-            return GetAsyncEncryptor(null);
-        }
-
-        /// <summary>
-        /// Gets an instance of <see cref="IAsyncEncryptor"/> for the provided encrypt key.
+        /// Gets an instance of <see cref="IAsyncEncryptor"/> for the provided key identifier.
         /// </summary>
         /// <param name="keyIdentifier">
         /// An implementation-specific object used to identify the key for this
         /// encryption operation.
         /// </param>
         /// <returns>An object that can be used for encryption operations.</returns>
-        public static IAsyncEncryptor GetAsyncEncryptor(object keyIdentifier)
+        public static IAsyncEncryptor GetAsyncEncryptor(object keyIdentifier = null)
         {
             return Current.AsAsync().GetAsyncEncryptor(keyIdentifier);
         }
 
         /// <summary>
         /// Asynchronously gets an instance of <see cref="IAsyncDecryptor"/> for the provided
-        /// encrypt key.
-        /// </summary>
-        /// <returns>An object that can be used for decryption operations.</returns>
-        public static IAsyncDecryptor GetAsyncDecryptor()
-        {
-            return GetAsyncDecryptor(null);
-        }
-
-        /// <summary>
-        /// Asynchronously gets an instance of <see cref="IAsyncDecryptor"/> for the provided
-        /// encrypt key.
+        /// key identifier.
         /// </summary>
         /// <param name="keyIdentifier">
         /// An implementation-specific object used to identify the key for this
         /// encryption operation.
         /// </param>
         /// <returns>An object that can be used for decryption operations.</returns>
-        public static IAsyncDecryptor GetAsyncDecryptor(object keyIdentifier)
+        public static IAsyncDecryptor GetAsyncDecryptor(object keyIdentifier = null)
```

Comment:
Is having a null keyId a valid scenario?  I know the change was to match the other methods.  But will this cause an error of any sort?

Reply:
Passing null is expected behavior. Doing so indicates that the "default" encryptor/decryptor should be used.

Reply:
Works for me.